<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="txt_pregunta">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>541</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>PREGUNTA</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>711</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Opciones:</string>
    </property>
    <widget class="QRadioButton" name="rb_opcionA">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Opcion A</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_opcionB">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Opcion B</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_opcionC">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Opcion C</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_opcionD">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>150</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Opcion D</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_validar">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>500</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>VALIDAR</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
